packages/debugger/lib/session/actions/index.js

```diff
@@ -1,9 +1,10 @@
 export const START = "SESSION_START";
-export function start(provider, txHash) {
+export function start(provider, txHash, ensRegistryAddress) {
   return {
     type: START,
     provider,
-    txHash //OPTIONAL
+    txHash, //OPTIONAL
+    ensRegistryAddress //OPTIONAL
```

Comment:
I think we're better-off switching this to an `ens` namespace also, following that change I suggest for `forTx` and `forProject`.

Sure, this is an internal concern, but seems like you might want to be able to add properties here just the same as you would anywhere.

Reply:
Doing that in an action seems a little awkward, but I guess so?